### PR TITLE
Katello: Update pulp_deb gem dependency for rpm/3.8

### PR DESCRIPTION
**Pull request by quba42**
This belongs together with:

- https://github.com/Katello/katello/pull/10733
- https://github.com/theforeman/foreman-packaging/pull/9777

### CODE REVIEW

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -6,7 +6,7 @@
 %global prereleasesource rc2
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
 %global mainver 4.10.0
-%global release 2
+%global release 3
```
Comment (quba42) [inline packages/katello/rubygem-katello/rubygem-katello.spec:9]: Not sure if this is what needs to be bumped here.

**Review by ianballou — APPROVED** [review]
This is looking fine to me

**Comment by ianballou** [comment]
@Odilhao is this good to merge, or are you waiting to get closer to 4.10 GA ?

**Comment by pcreech** [comment]
[test rpm]

**Comment by quba42** [comment]
Looks like @pcreech cherry-picked the relevant change to rpm/3.8 independently of this PR. Closing.

**Pull request closed without merging**